templates/hpc/index.html

```diff
@@ -0,0 +1,717 @@
+{% extends "openstack/_base_openstack.html" %}

```

Comment:
The meta data of `_base_openstack.html` is for openstack pages so i think we might need a new base file for hpc pages.